#!/usr/bin/env python
PACKAGE = "biped_wbms_core"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hpf_cutoff_hz", double_t, 0, "hpf_cutoff_hz", 20.0, 0.01 , 1000.0)
gen.add("lpf_cutoff_hz", double_t, 0, "lpf_cutoff_hz", 0.3, 0.01 , 1000.0)
gen.add("gain", double_t, 0, "gain", 0.1, 0.0 , 1e3)
gen.add("hpf_gain", double_t, 0, "hpf_gain", 0.1, 0.0 , 1e3)
gen.add("lpf_gain", double_t, 0, "lpf_gain", 0.0, 0.0 , 1e3)
gen.add("rate", double_t, 0, "rate", 500.0, 0.01 , 1e3)

gen.add("step_height", double_t, 0, "rate", 0.07, 0.01 , 0.3)
gen.add("step_time", double_t, 0, "rate", 1.0, 0.8 , 2.0)

exit(gen.generate(PACKAGE, "biped_wbms_core", "BipedWbmsCore"))